src/Akeneo/Tool/Bundle/MeasureBundle/Resources/public/bridge/dependencies.ts

```diff
@@ -19,6 +20,9 @@ const dependencies = {
   },
   user: userContext.get.bind(userContext),
   notify: messenger.notify.bind(messenger),
+  security: {
+    isGranted: securityContext.isGranted.bind(securityContext),
+  },
```

Comment:
```suggestion
  security: securityContext.isGranted.bind(securityContext),
```
what do you think ? do we have something else in security ?

Reply:
don't know. Just copy/pasted from connectivity

Reply:
Ok will do!